src/components/MangaControl.tsx

```diff
@@ -42,6 +43,31 @@ type Props = {
 
 const MangaControl: React.FunctionComponent<Props> = (props: Props) => {
   const { readerSettings, updateReaderSettings, keyBindings, setKeyBindings, chapter, chapters, updateChapter, page, pages, updatePage } = props;
+
+  const [colourMode, setColourMode] = useDarkMode();
+  const colourThemeValues = [

```

Comment:
why are you doing this - don't you already have all the names of the themes from the json

Reply:
also either have the theme name in the json be the display name ie
```
{
  'Mizu': {...},
 'Oblivion Hagoromo': {...},
 ...
}
```
or add a `displayName field`